runtimevar/example_test.go

```diff
@@ -77,6 +77,20 @@ func ExampleVariable_Latest_stringVariable() {
 	// "hello world"
 }
 
+func Example_latestStringVariableHowto() {
```

Comment:
Why the `Howto` suffix?

Reply:
The previous example uses `constantvar` and has an actual output expectation. I don't want these to show up. The pattern of having 

```
var v *PortableType
```

Is already used elsewhere in examples, since we're only compiling them.

Reply:
What happens if you do the "variables set up elsewhere" above the `constantvar.NewBytes`?